spec/ddtrace/auto_instrument_spec.rb

```diff
@@ -100,6 +100,20 @@ def migrate_db
 
       expect(route_span).to_not have_error
     end
+
+    context 'when `DD_SERVICE_PREFIX` env variable is set' do
+      around do |example|
+        ClimateControl.modify('DD_SERVICE_PREFIX' => 'my_app-') do
+          example.run
+        end
+      end
+
+      it 'adds the prefix to all names of all services' do
+        is_expected.to be_ok
+
+        expect(sqlite_span.service).to eq('my_app-sqlite')
```

Comment:
This test is invalid since the span is linked with the service identifier, not the name. Shall I check out the service configuration in Datadog registry instead?